src/views/Home/components/Footer/index.view.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
Because we upgraded to a new version of typescript in #71, we no longer have to import `react` if we don't use it. Instead, we can import it using the following since this is the convention that it is moving towards using this newer version of react: 
`import * as React from "react";`

Reply:
Okay, cool ~ no problem! In that case, we might want to change this notation for all other files too!

Reply:
Yeah -- I believe I ran a command that changed all the existing import statements. This only appears once or twice in the codebase. I did a `cmd + F` on the existing codebase and it looks like I added the wrong import statement after the PR was merged in. I think we should address those in a separate PR. Thoughts?

Reply:
Yep! I was lowkey planning a separate PR to just clean up the codebase a little bit, so that sounds like a good idea!